TLM/TLM/Manager/Impl/VehicleBehaviorManager.cs

```diff
@@ -1509,7 +1481,7 @@ public class VehicleBehaviorManager : AbstractCustomManager, IVehicleBehaviorMan
 				return false;
 			}
 
-			uint vehicleRand = GetVehicleRand(vehicleId);
+			uint vehicleRand = GetStaticVehicleRand(vehicleId);
```

Comment:
This should not be here. It's part of another bugfix which is not related to the turn-on-red feature